core/flyout.js

```diff
@@ -774,6 +774,7 @@ Blockly.Flyout.prototype.onMouseDown_ = function(e) {
   this.dragMode_ = Blockly.DRAG_FREE;
   this.startDragMouseY_ = e.clientY;
   this.startDragMouseX_ = e.clientX;
+  Blockly.Flyout.startFlyout_ = this;
```

Comment:
This line is necessary at the moment because Flyout.terminateDrag_ uses startFlyout_ to reset the flyout's dragMode_. Is there an alternative?